packages/cashier/src/components/cashier-onboarding/cashier-onboarding-side-note.tsx

```diff
@@ -26,7 +39,7 @@ const CashierOnboardingSideNote = ({ currency, is_crypto, openRealAccountSignup,
                     <span
                         key={0}
                         className='link link--orange cashier-onboarding-side-note__text-nowrap'
-                        onClick={() => window.LC_API.open_chat_window()}
+                        onClick={() => (window as any).LC_API.open_chat_window()}
```

Comment:
@vinu-deriv Would be better to declare the types in a declaration file in my opinion 🤔 

```ts
declare global {
    interface Window { 
        LC_API: {
            open_chat_window: VoidFunction;
        };
    }
}
```